src/runner/browser-env/vite/socket.ts

```diff
@@ -104,6 +104,12 @@ function handleBrowserEvents(
         io.to(runUuid).except(socket.id).emit(BrowserEventNames.initialize, payload);
     });
 
+    socket.on(BrowserEventNames.callConsoleMethod, payload => {
+        const { runUuid } = socket.handshake.auth;
+
+        io.to(runUuid).except(socket.id).emit(BrowserEventNames.callConsoleMethod, payload);
```

Comment:
I could call the console log right here (in master), but to my mind it's better to call it in the worker, that runs the test